### DISCUSSION TITLE
Why YAPF has been chosen over black?

### DISCUSSION THREAD

**Discussion by dekoza** [collerek/ormar#1138] [General]
I'm wondering why the [inferior](https://news.ycombinator.com/item?id=17155048) formatter has been chosen. I thought the Python community as a whole already embraced `black` as THE formatter for Python code by now.

**Comment by collerek** [comment]
Ormar is using black (with checks both in `pre-commit` and GitHub actions). Yapf config is for mkdocstrings for documentation.
